### DISCUSSION TITLE
Spam Attack Limit mode moderation option

### DISCUSSION THREAD

**Discussion by p37307** [mastodon/mastodon#28421] [General]
It would be great if federation moderation had a "_spam attack limit_' option to limit other instances during a time when that other instance is under a spam assault and ends up being the host of automated spammers, sending spam to users across the fediverse through bot accounts on their instance. It would work like the current limit mode does but expire after a certain amount of time. This way, an admin doesn't have to go back into moderation and remove the limit block.

It's only time until spammers start hitting multiple instances of various sizes at once instead of just the bigger ones.  Having to limit and then remove the limit, some innocent instances could get limited forever by oversight.

**Comment by StrangeMinds1960** [comment]
Unless I've misunderstood your post, you seem to be under the assumption that limit only works on instances.  Both limit and suspend can be applied to individual accounts as well

**Comment by p37307** [comment]
> Unless I've misunderstood your post, you seem to be under the assumption that limit only works on instances. Both limit and suspend can be applied to individual accounts as well

I'm not talking about normal spam. I am talking about when an instance gets under attack and starts sending out voluminous numbers of spam posts.

I didn't explain it correctly. I adjusted it. When another instance becomes host to spammers and they end up spamming, esp Private Mention spams, through automated posts from accounts on their instance, we could turn on the spam attack limit on our instances, which would expire at some point, well after they take the steps to figure out how to block whatever is creating the accounts and sending out the voluminous spam emails. 

Several of the larger instances have had private mention posting volleys in recent days. Admins that limited those instances will have to go back and unlimit them
